pkg/graphql/spawn.go

```diff
@@ -223,3 +226,22 @@ func (spawner *Spawner) BuildPodSpec(podSpec *corev1.PodSpec) {
 	}
 
 }
+
+func mountEmptyDir(podSpec *corev1.PodSpec, containers []*corev1.Container, name string, path string) {
+	emptyDirLimit := resource.MustParse(viper.GetString("jobSubmission.workingDirSize"))
```

Comment:
We can use `resource.ParseQuantity` to check if the setting is in correct format in `loadConfig` of `main.go` beforehand.

So that we won't crash in the middle if this settings is incorrect.